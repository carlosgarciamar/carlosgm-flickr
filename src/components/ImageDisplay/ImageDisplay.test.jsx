import React from 'react';
import { create } from 'react-test-renderer';

describe('<ImageDisplay />', () => {
  const props = {
    title: 'Photo title',
    author: 'author',
    photoUrl: 'http://www.example.com/photo-url',
    authorUrl: 'http://www.example.com/author-url',
    description: 'Lorem ipsum',
    tags: 'tag1 tag2',
  };
});
