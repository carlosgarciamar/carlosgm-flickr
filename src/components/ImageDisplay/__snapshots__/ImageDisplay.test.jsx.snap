// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ImageDisplay /> should match its snapshot 1`] = `
<div
  className="wrapper"
>
  <img
    alt="Photo title"
    className="image"
    src="http://www.example.com/photo-url"
  />
  <div
    className="textWrapper"
  >
    <a
      href="http://www.example.com/photo-url"
      target="__blank"
    >
      Photo title
    </a>
     by 
    <a
      href="http://www.example.com/author-url"
      target="__blank"
    >
      author
    </a>
    <div
      className="description"
    >
      Lorem ipsum
    </div>
    <div
      className="tags"
    >
      tag1, tag2
    </div>
  </div>
</div>
`;
